src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2841,6 +2841,33 @@
     /**
      * The {@code &&} (conditional AND) operator.
      *
+     * <p>For example:</p>
+     * <pre>
+     * if (true || true)
+     * {
+     * message = "Returns True";
```

Comment:
actaully we do not need statements inside scope and lets use variables instead of `true` literals

smth like
```
if (a || b) {
}
```
is enough as an example, please do not forget to update AST tree as well

Reply:
<pre>
<code>
if( a || b){
}
</code>
</pre>

Only this much changes? No message required under {....} ?

Reply:
Ok got it :)